docs/AnaplanExport-batchsource.md

```diff
@@ -148,8 +73,6 @@ corresponding CDAP data type for each Anaplan type.
 | text           | string         |
 | bool           | boolean        |
 | date           | date           |
-| date           | timestamp      |
-| date           | datetime       |
 | numeric        | decimal        |
 | numeric        | double / float |
```

Comment:
put double, float, int, long on their own rows.

Also, what does it mean that the Anaplan numeric type maps to multiple types? Shouldn't it map to a specific CDAP type?

Or is it something weird where it can be an int or a double depending on how people use it?

Reply:
oh, updated.

There is only a numeric type in the anaplan, and it can be either decimal or integer and they can be in the same column.

So I would like to show the idea that, user can transfer this numeric type data into whatever CDAP number type according to the real use case.

Reply:
I see. It's ok to have this then, but it should be documented above.

Also, have you tested that these all work? I'm pretty sure trying to read csv/tsv into a decimal will fail.